src/usr/local/buildpack/tools/v2/maven.sh

```diff
@@ -9,6 +9,9 @@ function install_tool () {
   local versioned_tool_path
   versioned_tool_path=$(create_versioned_tool_path)
 
+  local checksum_file
+  local original_checksum
+
```

Comment:
unrelated change?

Reply:
yeah, I unified the changes, so that the naming is consistent between each tool when we fetch the checksums

Reply:
i don't see usage of those new variables. please create a refactor PR first.